get-config-state.js

```diff
@@ -40,6 +40,10 @@ function getConfigState(dirname, opts) {
         dc && NODE_ENV ?
             join(configFolder, NODE_ENV + '.' + dc.datacenter + '.json') :
             null,
+        // load ./config/secrets-NODE_ENV.json
+        NODE_ENV ? join(configFolder, 'secrets-' + NODE_ENV + '.json') : null,
```

Comment:
I think this makes it reasonable flexible. Depending how the secrets may get loaded in multiple environments there may be common secrets and ones per env. This also allows test secrets etc to just live as static files someplace.